src/Text/Pandoc/Writers/Docx/OpenXML.hs

```diff
@@ -171,6 +171,58 @@ makeTOC opts = do
       ])
     ]] -- w:sdt
 
+makeLOF :: (PandocMonad m) => WriterOptions -> WS m [Element]
+makeLOF opts = do
+  let lofCmd = "TOC \\h \\z \\t \"Image Caption\" \\c" :: Text
```

Comment:
Why "Image Caption"?

Reply:
See: https://github.com/jgm/pandoc/blob/edb9e049cae72084cf2cf5bdb6e982ee25a34f24/src/Text/Pandoc/Writers/Docx/OpenXML.hs#L440

tldr, the "Image Caption" style is used for figure captions in docx.